frontend/packages/kubevirt-plugin/integration-tests-cypress/view/snapshot.ts

```diff
@@ -0,0 +1,11 @@
+export const takeSnapshot = 'button[id="add-snapshot"]';
```

Comment:
Would you mind separating the functions from the constants and maybe add an empty line to make it easier to see that they're different? Could you also add `Button` or `Btn` to the end of the button selectors here that are missing it?